src/network/clientpackethandler.cpp

```diff
@@ -1383,6 +1383,17 @@ void Client::handleCommand_CSMRestrictionFlags(NetworkPacket *pkt)
 	loadMods();
 }
 
+void Client::handleCommand_PlayerSpeed(NetworkPacket *pkt)
+{
+	v3f added_vel;
```

Comment:
For faster visual reaction, this packet should better include the object ID so that the clients look up the GenericCAO and apply the velocity change. I.e. it's then needed to send the packet to all players.

Reply:
http://irc.minetest.net/minetest-dev/2019-08-10#i_5576125